great-idea/css/index.css

```diff
@@ -62,84 +62,98 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 15px;
 }
-
-nav {
-	display: inline-block;
-	vertical-align: top;
-	word-spacing: 200px;
-	position: relative;
-	left: 50px;
-	top: 30px;
-}
-
-.logo {
-	display: inline-block;
-	vertical-align: top;
-	margin: 20px;
-	position: relative;
-	top: 0px;
-	left: 150px;
-}
-.main-content {
-	border-top: solid black 1px;
-	border-bottom: solid black 1px;
-	margin: 30px;
-}
-.cta-text {
-	display: inline-block;
-	width: 50%;
-	font-size: 75px;
-	position: relative;
-	left: 450px;
-	bottom: 10px;
-}
-.cta-text h1 br {
-	display: inline-block;
-	position: relative;
-	left: 100px;
-}
-.cta img {
-	display: inline-block;
-	margin: 20px;
-	position: relative;
-	top: 20px;
-	left: 300px;
-}
-.top-content {
-	column-count: 2;
-	display: inline-block;
-	vertical-align: top;
-	margin: 15px;
-	width: 50%;
-	position: relative;
-	right: 15px;
-}
-.middle-img {
-	display: inline-block;
-	vertical-align: top;
-	margin: 15px;
-	width: 50%;
-	position: relative;
-	right: 15px;
-}
-.bottom-content {
-	column-count: 3;
-	display: inline-block;
-	vertical-align: bottom;
-	margin: 30px;
-	width: 50%;
-	position: relative;
-	right: 30px;
-}
-.contact {
-	display: inline-block;
-	vertical-align: top;
-	position: relative;
-	left: 30px;
+.container {
+	margin: 0 auto;
+	margin-left: 200px;
+	margin-right: 200px;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	align-content: space-between;
+}
+.container header {
+	display: flex;
+	justify-content: space-around;
+	align-items: flex-end;
+}
+.container header nav {
+	width: 75%;
+	display: flex;
+	justify-content: space-evenly;
+	align-items: center;
+}
+.container header nav a {
+  font-size: 14px;
+  color: gray;
+  text-decoration: none;
+}
+.container header img {
+	margin-top: 20px;
+	align-self: center;
+}
+.container .cta {
+	display: flex;
+	justify-content: space-evenly;
+}
+.container .cta .cta-text {
+	margin-top: 120px;
+	font-size: 77px;
+    text-align: center;
+	display: flex;
+	flex-direction: column;
```

Comment:
One thing I'd like you to think about is making use of inheritance. For example, `.container .cta .cta-text` and `.container .main-content` both include some of the same attributes. If you can generalize these as a `.container` class (and override the values as necessary for elements that don't follow this pattern), it will make your code more efficient and easier to maintain.